src/Messages/OWSMessageSender.m

```diff
@@ -607,11 +630,31 @@ - (void)groupSend:(NSArray<SignalRecipient *> *)recipients
     [self saveGroupMessage:message inThread:thread];
     NSMutableArray<TOCFuture *> *futures = [NSMutableArray array];
 
+    NSArray<NSString *> *blockedPhoneNumbers = _blockingManager.blockedPhoneNumbers;
+    int blockedCount = 0;
     for (SignalRecipient *rec in recipients) {
-        // we don't need to send the message to ourselves, but otherwise we send
-        if (![[rec uniqueId] isEqualToString:[TSStorageManager localNumber]]) {
-            [futures addObject:[self sendMessageFuture:message recipient:rec thread:thread]];
+        // We don't need to send the message to ourselves...
+        if ([[rec uniqueId] isEqualToString:[TSStorageManager localNumber]]) {
```

Comment:
style nit: prefer `object.propertyName` for getters/setters, e.g. `rec.uniqueId`